netlify.toml

```diff
@@ -0,0 +1,12 @@
+[Settings]
+  ID = "ism-watershed-wellness-snapshot"
+
+[build]
+  base    = "./src/app/"
+  publish = "./src/app/build/"
+  command = "pushd ../.. && ./scripts/cibuild && popd"
+
+[[redirects]]
+  from = "/*"
+  to = "/index.html"
+  status = 200
```

Comment:
IDB's configuration has some additional declarations around headers and content types: https://github.com/azavea/idb-osm-extraction-tool/blob/develop/netlify.toml#L16

Consider adding those here.

Reply:
Thanks. Will do.